client/pages/index.tsx

```diff
@@ -55,16 +57,37 @@ const Home = () => {
       } else {
         console.log('No FormData generated by takePhoto.');
       }
+
+      // ********************************************************************
+      // NOTE: Nov 17
+      // added by victor to send image to GPT service
+      const gptResponse = await fetch('https://192.168.2.16:8080/gpt/ask', {
```

Comment:
be sure to use the environment variable for the server URL here, and instead of called gpt/ask we want to call the new endpoint from eri /api/request, which uses the new multer middleware that saves the img to disk and also converts the img into a buffer!
```suggestion
      const gptResponse = await fetch(`${process.env.SERVER_BASE_URL}/api/request`, {
```

Reply:
and another thing we would want to do is move all this code into a service similar to `client\services\cameraService.ts` we can do this in a separate issue though (not a priority)